namespaces/cloud-platform-live-0.k8s.integration.dsd.io/prisoner-categorisation-tool-dev/00-namespace.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: prisoner-categorisation-tool-dev
+  labels:
+    cloud-platform.justice.gov.uk/business-unit: HMPPS
+    cloud-platform.justice.gov.uk/is-production: false
```

Comment:
This will not work - it needs to be a string. This is an oversight in our templates (fix in #186). Can you please quote the labels and annotations?